### ISSUE TITLE
goocanvas is not findable

### ISSUE THREAD

**Issue by OzarkShepherd** [gramps-project/flatpak#2]
When I had goocanvas 2 in the flatpak, graphviz could not find goocanvas 2.

**Comment by Nick-Hall** [comment]
Is this an error from a graph report?  It may be looking to run _graphviz_ from the command line.

**Comment by OzarkShepherd** [comment]
No, it was an error that popped up when I started Gramps due to the GraphView addon.  Before the flatpak, I had the GraphView add-on installed on my prior Gramps installation.  So when I made the flatpak and ran it, Gramps opened up my genealogy file and prior Gramps settings.  Since Graphview couldn't find goocanvas 2, it threw the error at startup about not finding goocanvas 2.  However, I had goocanvas2, Graphviz, and pygraphviz installed in the flatpak at that time.  So what I did was remove GraphView from my addons, kept graphviz in the flatpak, and took out goocanvas 2 and pygraphviz to possibly deal with later since they were optional dependencies.

**Comment by OzarkShepherd** [comment]
I've revisited Goocanvas 2 for the past few days in my testing fork, and still have the same problem of Goocanvas 2 (and 3) not being found.  The picture shows the result of running the Gramps flatpak from terminal in my VM with GooCanvas installed.  I'm wondering if the canberra-gtk-module error is a related issue, but internet searches show it has been around for a while with other flatpaks.
![Screenshot from 2021-02-11 13-35-08](https://user-images.githubusercontent.com/68165701/107824890-cb8c3080-6d47-11eb-9e32-2cdcf0fc592c.png)
I tried installing Gnome gobject introspection as a substitute for gir1.2-goocanvas-2.0, for which I couldn't find an archive that isn't a deb file.  The result was that the Gramps flatpak stopped launching the GUI.  The dependencies mentioned should be part of the Gnome Platform that gets installed with the flatpak, but they are not being found.
![Screenshot from 2021-02-10 12-55-37](https://user-images.githubusercontent.com/68165701/107825460-c7144780-6d48-11eb-8f03-af6502e966bb.png)
So that is the current status of GooCanvas in the flatpak.
